### ISSUE TITLE
NgNonBindable no longer working as standalone injection

### ISSUE THREAD

**Issue by thelgevold** [angular/angular#4499]
I just noticed that injecting NgNonBindable no longer works if I include it as an individual directive. 
I get the following error:

Error: Unexpected directive value 'undefined' on the View of component 'IgnoreBindings'
    at new BaseException (http://localhost:63342/angular2-poc/node_modules/angular2/bundles/angular2.js:15158:21)
    at RuntimeMetadataResolver.getViewDirectivesMetadata (http://localhost:63342/angular2-poc/node_modules/angular2/bundles/angular2.js:16930:17)
    at RuntimeCommandFactory.componentTemplateFactory (http://localhost:63342/angular2-poc/node_modules/angular2/bundles/angular2.js:22911:66)
    at RuntimeCommandFactory.createBeginComponent (http://localhost:63342/angular2-poc/node_modules/angular2/bundles/angular2.js:10223:33)
    at CommandBuilderVisitor.visitElement (http://localhost:63342/angular2-poc/node_modules/angular2/bundles/angular2.js:10344:46)
    at ElementAst.visit (http://localhost:63342/angular2-poc/node_modules/angular2/bundles/angular2.js:9792:22)
    at http://localhost:63342/angular2-poc/node_modules/angular2/bundles/angular2.js:9870:27
    at Array.forEach (native)
    at Object.templateVisitAll (http://localhost:63342/angular2-poc/node_modules/angular2/bundles/angular2.js:9869:10)
    at CommandBuilderVisitor.visitElement (http://localhost:63342/angular2-poc/node_modules/angular2/bundles/angular2.js:10349:24)

However, it works if I include it as part of CORE_DIRECTIVES

Is NgNonBindable deprecated as a standalone directive?

Basically 
```
import {Component, View, CORE_DIRECTIVES} from 'angular2/angular2'; //works
import {Component, View, NgNonBindable} from 'angular2/angular2'; //resolved to undefined in the view
```

**Comment by pkozlowski-opensource** [comment]
@thelgevold see https://github.com/angular/angular/blob/09531d89868007eb0d62fc9ccd6d0836e4587809/CHANGELOG.md#compiler

I've sent a PR to update the changelog as we've missed this breaking change from it.

**Comment by thelgevold** [comment]
@pkozlowski-opensource Ah, thanks for clarifying
